apps/src/applab/applab.js

```diff
@@ -1480,6 +1490,16 @@ Applab.penColor = function (opts) {
   }
 };
 
+Applab.speed = function (opts) {
+  if (opts.percent >= 0 && opts.percent <= 100) {
```

Comment:
yeah, it's really pending our proper parameter validation work. (probably message out to console.log). For now, I just wanted to make sure that we didn't break the IDE.